test/unit/lists-component.spec.ts

```diff
@@ -97,16 +97,22 @@ describe('when getting picks from a list by list id', () => {
           userAddress: '0xuseraddress'
         })
       ).resolves.toEqual(dbGetPicksByListId)
+
       expect(dbQueryMock).toBeCalledWith(
         expect.objectContaining({
-          text: expect.stringContaining('WHERE list_id = $1 AND user_address = $2'),
-          values: expect.arrayContaining(['list-id', '0xuseraddress'])
+          strings: expect.arrayContaining([
+            expect.stringContaining('WHERE p.list_id = '),
+            expect.stringContaining(' AND (p.user_address = '),
+            expect.stringContaining(' OR favorites.acl.grantee = '),
+            expect.stringContaining(" OR favorites.acl.grantee = '*'")
```

Comment:
The test may fail because of the change in storing the `*` symbol in a variable.

WDYT?
```suggestion
            expect.stringContaining(' OR favorites.acl.grantee = ')
```